src/ray/internal/internal.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#include "ray/common/buffer.h"
+#include "ray/common/id.h"
+#include "ray/core_worker/common.h"
+#include "ray/stats/metric.h"
+
```

Comment:
Can you add comments to the code saying where it's used so people don't accidentally delete it again in the future?

Reply:
> Can you add comments to the code saying where it's used so people don't accidentally delete it again in the future?

Comments had been added in internal header file.